app/views/posts/index.html.erb

```diff
@@ -1 +1,18 @@
-<h2>Here is a list of posts for a given user</h2>
\ No newline at end of file
+<h1>User ID: <%= @user.id %> </h1>
+<h1>Name: <%= @user.name %> </h1>
```

Comment:
- [ ] Same here, Please change the second `<h1>` tag for semantic reasons.

Reply:
@Omar-Muhamad Thank you for reviewing my project, I will make the necessary corrections as suggested